src/client/components/ReviewEditorComponent/ReviewEditorComponent.style.css

```diff
@@ -0,0 +1,66 @@
+.container {
+  max-width: 598px;
```

Comment:
```suggestion
  width: 100%;
```

Reply:
To help and make it more responsive, given that we don't know yet how much space will be taken by the other elements in the same page where this will be use